src/utils/mergeOptions.ts

```diff
@@ -229,7 +229,18 @@ function getOutputOptions(
 	command: GenericConfigObject = {}
 ): OutputOptions {
 	const getOption = createGetOption(config, command);
-	const format = getOption('format');
+	let format = getOption('format');
+
+	// Handle format aliases
+	switch (format) {
+		case 'esm':
+		case 'module':
+			format = 'es';
+			break;
+		case 'commonjs':
+			format = 'cjs';
+			break;
+	}
```

Comment:
Would be nice to also get rid of the fallback in line 259.

Reply:
Thanks, done.